sechub-shared-kernel/src/main/java/com/mercedesbenz/sechub/sharedkernel/usecases/user/execute/UseCaseUserMarksFalsePositives.java

```diff
@@ -15,12 +15,12 @@
 @Target(ElementType.METHOD)
 @Retention(RetentionPolicy.RUNTIME)
 @UseCaseDefinition(
-		id=UseCaseIdentifier.UC_USER_MARKS_FALSE_POSITIVES_FOR_FINISHED_JOB,
+		id=UseCaseIdentifier.UC_USER_MARKS_FALSE_POSITIVES,
 		group=UseCaseGroup.SECHUB_EXECUTION,
-		apiName="userMarksFalsePositivesForJob",
-		title="User marks false positives for finished sechub job",
-		description="user/mark_false_positives_for_job.adoc")
-public @interface UseCaseUserMarksFalsePositivesForJob {
+		apiName="userMarksFalsePositives",
+		title="User marks false positives",
+		description="user/mark_false_positives.adoc")
```

Comment:
The full Build is currently red because the file mark_false_positives.adoc does not exist. Please make sure to write the documentation to the missing file.